easybuild/easyblocks/g/gromacs.py

```diff
@@ -201,9 +203,20 @@ def configure_step(self):
                 # to avoid that GROMACS finds and uses a system-wide CUDA compiler
                 self.cfg.update('configopts', "-DGMX_GPU=OFF")
 
-        # check whether PLUMED is loaded as a dependency
+        # PLUMED detection
+        # enable PLUMED support if PLUMED is listed as a dependency
+        # and PLUMED support is either explicitly enabled (plumed = True) or unspecified ('plumed' not defined)
         plumed_root = get_software_root('PLUMED')
+        if self.cfg['plumed'] and not plumed_root:
+            msg = "The PLUMED module needs to be loaded to build GROMACS with PLUMED support."
```

Comment:
msg = "PLUMED is not listed as a dependency but PLUMED support has been requested."
or
msg = "PLUMED support has been requested but PLUMED is not listed as a dependency."

I.e. it's not about the PLUMED module as such since it may not exist when starting the GROMACS build and it should not be loaded manually before running the eb command.

Reply:
Thanks. Indeed your suggestions are more clear.  
I went with the second one.